### PR TITLE
[14.0][ADD] repair_stock_move (Alpha)

### PR DESCRIPTION
**Pull request by MateuGForgeFlow**
**Changes the repair module flow**

Modifies the creation of stock moves in the repair flow:
- Confirm Repair: creates draft stock moves on confirmation of the repair order.
- Start Repair: sets the stock move state to "Waiting Availability".
- End Repair: completes the stock moves and ends the repair order.

### CODE REVIEW

File: repair_stock_move/README.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+Authors
+~~~~~~~
+
+* ForgeFlow S.L.
+
+Contributors
+~~~~~~~~~~~~
+
+* Jordi Ballester Alomar <jordi.ballester@forgeflow.com>
```
Comment (JordiBForgeFlow) [inline repair_stock_move/README.rst]: So many people involved??

**Comment by marcelsavegnago** [comment]
If possible, when it is merged I suggest doing it for V12 and V13 as well. 😅

**Comment by marcelsavegnago** [comment]
https://github.com/odoo/odoo/pull/66496

**Comment by marcelsavegnago** [comment]
@MateuGForgeFlow I recently added 2 new modules to improve the repair module. Base_repair is already at 13 and I can migrate to 14 as well. Base_repair_config is still awaiting approval for 13. They may be useful if you need to.

https://github.com/OCA/manufacture/tree/12.0/base_repair
https://github.com/OCA/manufacture/tree/12.0/base_repair_config

Need review https://github.com/OCA/manufacture/pull/601

**Review by LoisRForgeFlow — APPROVED** [review]
I think this is ready for now considering that it is an Alpha version.

**Comment by TobiasClemens** [comment]
> I think this is ready for now considering that it is an Alpha version.

Hello, 
is there a timetable when the Production Version is ready?
Thank you!

**Pull request merged**